lib/src/rendering/functions/cancel.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:ui';
+
+import 'package:catex/src/lookup/context.dart';
+import 'package:catex/src/rendering/rendering.dart';
+
+/// [RenderNode] for [CancelNode].
+class RenderCancel extends RenderNode {
+  /// Constructs a [RenderCancel] given a [context].
+  RenderCancel(CaTeXContext context) : super(context);
+
+  Paint _linePaint;
+
+  @override
+  void configure() {
+    final childSize = sizeChildNode(children[0]);
+
+    renderSize = childSize;
+
+    _linePaint = Paint()
+      ..strokeWidth = 2
+      ..color = context.color;
+  }
+
+  @override
+  void render(Canvas canvas) {
+    paintChildNode(children[0]);
+    final double horizontalPadding = renderSize.width * .05;
+    final double verticalPadding = renderSize.height * .05;
```

Comment:
We use implicit types.

```suggestion
    final horizontalPadding = renderSize.width * .05;
    final verticalPadding = renderSize.height * .05;
```